Makefile

```diff
@@ -30,23 +26,29 @@ ifeq ($(UNAME_S),Darwin)
 	MOBY_DOCKER=/Applications/Docker.app/Contents/Resources/bin/docker
 endif
 
-BINARY_FOLDER=$(shell pwd)/bin
-GIT_TAG?=$(shell git describe --tags --match "v[0-9]*")
 TEST_FLAGS?=
 E2E_TEST?=
 ifeq ($(E2E_TEST),)
 else
 	TEST_FLAGS=-run $(E2E_TEST)
 endif
 
-all: compose-plugin
+BUILDX_CMD ?= docker buildx
 
-.PHONY: compose-plugin
-compose-plugin: ## Compile the compose cli-plugin
+all: build
+
+.PHONY: build ## Build the compose cli-plugin
+build:
+	$(eval DESTDIR ?= $(shell pwd)/bin/build)
+	mkdir -p "$(DESTDIR)"
+	CGO_ENABLED=0 GO111MODULE=on go build -trimpath -tags "$(GO_BUILDTAGS)" -ldflags "$(GO_LDFLAGS)" -o "$(DESTDIR)/docker-compose" ./cmd
```

Comment:
Was there a specific reason to require an absolute path for `DESTDIR`?

I think the `mkdir` may be redundant as well; `go build` should create missing directories. I just gave that a try to be sure, but it looks to be doing that;

```bash
ls somedir
ls: somedir: No such file or directory

go build -o somedir/foo .

ls somedir/
foo*
```